### PR TITLE
Feature/split deployment

### PR DESCRIPTION
**Pull request by ekwanNTT**
# Description

This PR includes the following proposed change(s):

- {List all the changes, if possible add the jira ticket #}

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Refactoring / Documentation

if your change is a breaking change, please add `breaking change` label to this PR

## How Has This Been Tested?

Please describe the tests that you ran to verify your changes.

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: jag-ccd-application-1.1/src/main/resources/logback.xml [hunk 1]
```diff
@@ -18,7 +18,7 @@
         <url>${SPLUNK_HTTP_URL}</url>
         <token>${SPLUNK_TOKEN}</token>
         <index>${SPLUNK_INDEX}</index>
-        <source>ccd-application</source>
+        <source>ccd-application-11</source>
```
Comment (sylar-zeyangz) [inline jag-ccd-application-1.1/src/main/resources/logback.xml:21]: shouldn't it be ccd-application-1.1?

File: jag-ccd-application-1.2/src/main/java/ca/bc/gov/open/ccd/configuration/SoapConfig.java [hunk 2]
```diff
@@ -0,0 +1,129 @@
...
+        SaajSoapMessageFactory messageFactory = new SaajSoapMessageFactory();
+        messageFactory.setMessageProperties(props);
+        messageFactory.setSoapVersion(SoapVersion.SOAP_12);
+        return messageFactory;
+    }
+
+    @Bean(name = "CCD.Source.CivilFileContent.ws:CivilFileContentSecure")
+    public Wsdl11Definition civilFileContentSecureWSDL() {
+        SimpleWsdl11Definition wsdl11Definition = new SimpleWsdl11Definition();
+        wsdl11Definition.setWsdl(new ClassPathResource("xsdSchemas/civilFileContentSecure.wsdl"));
```
Comment (sylar-zeyangz) [inline jag-ccd-application-1.2/src/main/java/ca/bc/gov/open/ccd/configuration/SoapConfig.java:83]: Still 11 Definition?

**Review by sylar-zeyangz — CHANGES_REQUESTED** [review]
Other issues 
- Secure endpoints were not removed from ccd-application-1.1 controllers
- Unit tests are working but do we need to migrate tests for secured endpoints to 1.2?

File: jag-ccd-application-1.1/src/main/resources/logback.xml (hunk 1 above)
Comment (ekwanNTT) [reply jag-ccd-application-1.1/src/main/resources/logback.xml:21]: Dot might get weird here

File: jag-ccd-application-1.2/src/main/java/ca/bc/gov/open/ccd/configuration/SoapConfig.java (hunk 2 above)
Comment (ekwanNTT) [reply jag-ccd-application-1.2/src/main/java/ca/bc/gov/open/ccd/configuration/SoapConfig.java:81]: Yep.

**Review by sylar-zeyangz — APPROVED** [review]

**Pull request merged**